test/WebJobs.Extensions.DurableTask.Analyzers.Test/Binding/ClientAnalyzerTests.cs

```diff
@@ -61,26 +73,35 @@ public static class HireEmployee
     {
         [FunctionName(""HireEmployee"")]
         public static async Task<Application> RunOrchestrator(
-            [DurableClient] IDurableClient durableClient, [DurableClient] IDurableOrchestrationClient durableOrchestrationClient, 
-            [DurableClient] IDurableEntityClient durableEntityClient, ILogger log)
+            [DurableClient] IDurableEntityClient client,
+            ILogger log)
             {
             }
 }";
+
+        [TestMethod]
+        public void DurableClient_V1_NonIssue()
```

Comment:
I'm not sure I understand what is meant  by the  `_NonIssue` suffix. From what I can tell, this test verifies that the `v1Fix` code does not create any warnings, but that is not immediately clear from the test name.